src/query/src/promql/planner.rs

```diff
@@ -941,11 +941,15 @@ impl PromPlanner {
     }
 
     // TODO(ruihang): ignore `MetricNameLabel` (`__name__`) matcher
-    fn matchers_to_expr(&self, label_matchers: Matchers) -> Result<Vec<DfExpr>> {
+    fn matchers_to_expr(label_matchers: Matchers) -> Result<Vec<DfExpr>> {
         let mut exprs = Vec::with_capacity(label_matchers.matchers.len());
         for matcher in label_matchers.matchers {
             let col = DfExpr::Column(Column::from_name(matcher.name));
-            let lit = DfExpr::Literal(ScalarValue::Utf8(Some(matcher.value)));
+            let unescaped_value: String =
```

Comment:
I think if unescaping the value fails, we must fallback to the raw value.

Looks like grafana only escaping the template variables in PromQL https://github.com/grafana/grafana/issues/4234

Reply:
resolved in [e014c61](https://github.com/GreptimeTeam/greptimedb/pull/5521/commits/e014c613c80d81c14795e1d1815fb0ddb698ad0e)